tests/specs/graph/JsrSpecifiers_FastCheck_ClassPropertiesLeavable.txt

```diff
@@ -0,0 +1,120 @@
+# https://jsr.io/@scope/a/meta.json
+{"versions": { "1.0.0": {} } }
+
+# https://jsr.io/@scope/a/1.0.0_meta.json
+{ "exports": { ".": "./mod.ts" } }
+
+# https://jsr.io/@scope/a/1.0.0/mod.ts
+import type { Foo, Bar } from "./b.ts";
+
+export class Foo {
+  bar = (a: string) => "test";
+  foo = () => "test" as Foo;
+  bar = (b: Bar) => 1;
```

Comment:
We should add a similar test for function expressions. Something like:

```ts
  baz = function(): PublicByBaz { return new PublicByBaz(); }
}

class PublicByBaz {}
```

Reply:
Done